packages/collector/test/tracing/messaging/bull/sender.js

```diff
@@ -0,0 +1,94 @@
+/*
+ * (c) Copyright IBM Corp. 2021
+ * (c) Copyright Instana Inc. and contributors 2021
+ */
+
+'use strict';
+
+require('../../../../')();
+const logPrefix = `Bull (${process.pid}):\t`;
+const Queue = require('bull');
+const redisServer = process.env.REDIS_SERVER || 'redis://127.0.0.1:6379';
+const queueName = process.env.BULL_QUEUE_NAME || 'nodejs-team';
+const express = require('express');
+const port = process.env.APP_SENDER_PORT || 3215;
+const bullJobName = process.env.BULL_JOB_NAME || 'steve';
```

Comment:
I always meant to ask: Who is Steve? :-)

Reply:
well, it's a Job called Steve... got it?

Reply:
Ouch! 😂   Good one.